ffs/scheduler/internal/cistore/cistore.go

```diff
@@ -45,6 +51,67 @@ func (s *Store) Get(iid ffs.APIID, c cid.Cid) (ffs.StorageInfo, error) {
 	return ci, nil
 }
 
+// Query returns a list of StorageInfo matching any provided query options.
+func (s *Store) Query(iids []ffs.APIID, cids []cid.Cid) ([]ffs.StorageInfo, error) {
```

Comment:
New `cistore` method that lists all `StorageInfo`s filtered by api ids and/or cids.